package.json

```diff
@@ -42,37 +42,37 @@
     "prepare": "is-ci || husky install"
   },
   "dependencies": {
-    "@typescript-eslint/experimental-utils": "^5.5.0"
+    "@typescript-eslint/experimental-utils": "^5.9.0"
```

Comment:
Can you avoid upgrading this one? That would trigger a new patch version of the plugin.

Reply:
Since `v5.0.2` is released earlier today, we technically already use this version 🤷‍♂️

Reply:
We releasing a new version of the plugin doesn't mean the final user using our plugin will update to the very latest version of `@typescript-eslint/experimental-utils` when updating our plugin in their codebase.

We are defining a range of valid versions for `@typescript-eslint/experimental-utils` that the final user can match. The current range is equal to or greater than 5.5.0 before. A user with the current version of our plugin could have version 5.6.0 of `@typescript-eslint/experimental-utils`. They might have fixed their dependencies, so updating `eslint-plugin-testing-library` doesn't install all the latest versions of its dependencies (`@typescript-eslint/experimental-utils` in this case). They are free to remain using `@typescript-eslint/experimental-utils` 5.6.0 under the hood then.

Now you want to bump the valid range up to equal to or greater than 5.9.0, which means the previous user would get the sub dependency `@typescript-eslint/experimental-utils` forced to get bumped from 5.6.0 to 5.9.0. That's a change from the point of view of our Semantic Versioning Policy: there is a change for the final user, a sub dependency must be updated.

That's the reason why bumping `dependencies` for our plugin causes a new release of the plugin. Since this is a chore release, we don't need to upgrade our main dependencies.

Reply:
Let's make this a `fix` then 👍

Reply:
But why that rush to always set the very latest version?

I'm a bit worried about this, we are always mixing updating production dependencies and dev dependencies. We need to be a bit flexible with prod dependencies. Our plugin doesn't need the last version of that dependency, there is no benefit in doing this right now (and most of the times).

Reply:
I'm updating all the dependencies, so why not use the latest for normal dependencies too?

There's no harm in using the latest minor, as it's a not breaking anything at all 🤷‍♂️

Reply:
There is no harm for us, but there might be for some final users. Not the end of the world, of course. But in the same way that there is no harm, there is no benefit.

I'll approve this one just to get everything related to ESLint updated to v8, but please let's stick to the proper scope in future PRs when updating things.